Documentation/concepts/ebpf/maps.rst

```diff
@@ -36,7 +36,7 @@ Egress Policy            endpoint         16k             Max 16k endpoints acro
 For some BPF maps, the upper capacity limit can be overridden using command
 line options for ``cilium-agent``. A given capacity can be set using
 ``--bpf-ct-global-tcp-max``, ``--bpf-ct-global-any-max``,
-``--bpf-nat-global-max``, ``--bpf-neigh-global-max``, ``--bpf-policy-map-max``,
+``--bpf-nat-global-max``, ``--bpf-policy-map-max``,
```

Comment:
> Does this still mean its okay to disable the BPF map caches across the board ?

Reply:
Yeah, if there are clients in the same L2 which are not tracked on the LB / middle node's neighbor table, we do need the map indeed as otherwise replies for the snat mode will fail to populate the clients mac. Also discussed this with @brb today in the morning. I'm currently changing the neighbor map in the following way: i) we retain tracking client macs, ii) for backends the fib lookup will be used. On replies, we use the fib lookup and if it fails we can check the BPF neighbor map. So still the neighbor map function as a precache is removed. This will also implicitly solve @tklauser's remarks on the broken calculation for dynamic map sizing.